docs/0.4.x/en-US/quickstart.md

```diff
@@ -1,25 +1,31 @@
 
-This command has a few stages. First, there's *Generating your app...*, which will compile your app's *engine-side* (often called the *server-side*, but Perseus has exporting, tinkering, and a million other things that happen there, so we just call the not-browser the *engine* for simplicity) and build all your app's pages. Then, there's *Building your app to Wasm...*, which compiles your app's *browser-side* into WebAssembly, allowing it to be run in the browser. Finally, there's *Building server...*, which just compiles and prepares the server that Perseus will use to run your code. If it's the first time you're running Perseus, there'll also be a stage for *installing external tools...*, in which Perseus downloads some external dependencies, like `wasm-opt`, which helps to supercharge your Wasm in production. Here, Perseus will also use `rustup` to install the `wasm32-unknown-unknown` target, which is Rust's way of saying 'the browser'. If you're not using `rustup`, you'll need to install this target manually.
+   The `perseus serve` command involves several stages:
+   - **Generating your app**: Compiles the engine-side (server-side) of your app and builds all the app's pages.
+   - **Building your app to Wasm**: Compiles the browser-side of your app into WebAssembly (Wasm) for running in the browser.
+   - **Building server**: Compiles and prepares the server used by Perseus to run your code.
+   - **Installing external tools**: (for first-time Perseus users): Downloads external dependencies, such as `wasm-opt`, to enhance Wasm performance in production. Additionally, `rustup` installs the `wasm32-unknown-unknown` target, which represents the browser in Rust. If you're not using `rustup`, manually install this target.
 
-*In fact, all these stages run in parallel, which is why Perseus builds take up a fair bit of memory. If you're running on an older device, you might want to add the `--sequential` flag to the above command, which will run these steps one-by-one. This can also be useful if you're running multiple Perseus builds at once.*
+   **Note**: These stages run in parallel, requiring a significant amount of memory. If you're using an older device or running multiple Perseus builds simultaneously, consider using the `--sequential` flag with the `perseus serve` command to execute the stages sequentially.
 
-Now, if you change some code in, say, `my-app/src/templates/index.rs` (where, by convention, you store the code for your app's landing page), you should see the build process automatically restart, and, once it's done, your browser will automatically reload with your changes! 
+   Whenever you modify code in `my-app/src/templates/index.rs` (where the landing page code is conventionally stored), the build process will automatically restart, and your browser will reload with the updated changes.
```

Comment:
```suggestion
   Now, if you change some code in, say, `my-app/src/templates/index.rs` (where, by convention, you store the code for your app's landing page), you should see the build process automatically restart, and, once it's done, your browser will automatically reload with your changes!
```

Should make clear here that it will reload for any code changes, not just those to the index page.